.github/workflows/linux-ci.yml

```diff
@@ -13,14 +13,33 @@ on:
 jobs:
   build-test:
     runs-on: ubuntu-latest
+    env:
+      DOTNET_MULTILEVEL_LOOKUP: 1
 
     strategy:
       matrix:
-        version: [netcoreapp3.1,net6.0]
+        version: [netcoreapp3.1,net6.0,net7.0]
 
     steps:
     - uses: actions/checkout@v3
 
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '3.1.x'
```

Comment:
Note, 3.1 support is going to end by the end f this year. The change here is good, I wanted to mention this part may need to change in the near future.